lib/feature_management.rb

```diff
@@ -121,4 +121,8 @@ def self.allow_doc_auth_test_credentials?
   def self.email_addition_enabled?
     Figaro.env.email_addition_enabled == 'true'
   end
+
+  def self.backup_codes_2fa?
```

Comment:
I tested this locally. 
With `backup_codes_2fa: true` I observed the expected behavior when selecting backup codes as the first MFA option.

However, with `backup_codes_2fa: false` I did not observe the expected behavior. After selecting backup codes as the first MFA option I was forwarded to the Account page. I expected to see the backup MFA screen.

Reply:
Thank you @achapm .. investigating

Reply:
The issue resolved itself after a git pull.

I tested the with:
backup_codes_2fa: true 
AND
backup_codes_2fa: false

I observed the expected behaviour in both cases.